### PR TITLE
Skip reading ide-options-v2.json if doesn't exist to avoid throwing a…

### PR DESCRIPTION
**Pull request by lwronski**
…n error

### CODE REVIEW

File: modules/integration/docker/src/test/scala/scala/cli/integration/RunDockerTests.scala
Comment (MaciejG604) [inline modules/integration/docker/src/test/scala/scala/cli/integration/RunDockerTests.scala:1]: Those changes are unrelated, I think

**Review by MaciejG604 — APPROVED** [review]

**Review by tgodzik — APPROVED** [review]
Thanks for working on this!
